dev-python/cchardet/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="project">
```

Comment:
We [shouldn't](https://wiki.gentoo.org/wiki/Project:Python#Adding_packages) add python@ to projects unless you've agreed it with them (or any project).